game_event/models.py

```diff
@@ -51,3 +51,11 @@ def validate_game_event(court, ball_game, min_number_of_players, max_number_of_p
     @staticmethod
     def is_ball_game_playable_at_court(court, ball_game):
         return CourtBallGame.is_ball_game_playable(court, ball_game)
+
+    def delete_game_event(self):
```

Comment:
This new method is missing unit tests.

Reply:
i.e. we should try and test functions/methods directly, when possible. This will allow us the most flexibility for providing full test coverage.